src/Bookings.js

```diff
@@ -1,21 +1,52 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+//import "./Bookings.css";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      //    fetch("https://cyf-react.glitch.me/delayed")
+      //fetch("https://cyf-react.glitch.me/error")
+      .then(response => response.json())
+      .then(data => {
+        if (data.error) {
+          throw data;
+        }
+        console.log(data);
+        setBookings(data);
+        setLoading(false);
+      })
+      .catch(error => setError(error));
```

Comment:
Nice of use of `catch` - this will also trigger e.g. if you get a generic error with a non-JSON response, and the JSON parsing fails.

Reply:
Although in this case your `error.error` in the JSX below wouldn't work - one more reason to throw `data.error` instead of `data` above.